tests/UserDefinedFormTest.php

```diff
@@ -50,6 +50,26 @@ public function testGetCMSFields()
         $this->assertTrue($fields->dataFieldByName('OnCompleteMessage') != null);
     }
 
+	
+    public function testGetCMSFieldsShowInSummary() 
+    {
+        $this->logInWithPermission('ADMIN');
+        $form = $this->objFromFixture('UserDefinedForm', 'summary-rules-form');
+
+        $fields = $form->getCMSFields();
+
+        $this->assertTrue($fields->dataFieldByName('Submissions') != null);
```

Comment:
`$this->assertNotNull($fields->dataFieldByName('Submissiones'));`

Maybe even `$this->assertInstanceOf(GridField::class, $fields->dataFieldByName('Submissiones'));` would be appropriate?